apps/settings/locales/settings.en-US.properties

```diff
@@ -290,7 +290,13 @@ message-delivery-reports = Delivery Reports
 message-delivery-reports-details = Request notification of delivery for each message sent
 message-read-reports = Read Reports
 message-read-reports-details = Request read report for each message sent
-mmsTitle = Multimedia Messaging (MMS)
+# [For localizers]
+# We can't render these strings correctly for RTL locales, so we have to put LRE (&#x202a;)
+# and PDF (&#x202c;) entities to these translations to force the correct direction.
+#
+# RTL locales should have a translation similar to this:
+# mmsTitle.innerHTML= &#x202a;Multimedia Messaging (MMS)&#x202c;
+mmsTitle.innerHTML = Multimedia Messaging (MMS)
```

Comment:
I think you should also include LRE and PDF in the English copy here.  If the translation is not available, the build sytsem will copy the English value and without the Unicode marks the parentheses will look broken.